features/auto-detect-path.feature

```diff
@@ -10,7 +10,7 @@ Feature: Auto-detecting the path to the executable
 
   Scenario: relative path given
     Given I run the global command "node -h"
-    Then it prints "Usage: node [options]"
+    Then it prints "Usage: node \[options\]"
```

Comment:
Okay, I didn't consider that using a regex has such side effects on normal text. Let's go with your original idea of having two separate steps:

```cucumber
Then it prints /regex text/
Then it prints "normal text"
```

Then you can revert this change.